ddtrace/sampler.py

```diff
@@ -101,11 +102,11 @@ def __init__(self, sample_rate=1.0):
     def set_sample_rate(self, sample_rate):
         # type: (float) -> None
         self.sample_rate = float(sample_rate)
-        self.sampling_id_threshold = self.sample_rate * MAX_TRACE_ID
+        self.sampling_id_threshold = self.sample_rate * _MAX_UINT_64BITS
```

Comment:
are these changes all unnecessary since we're keeping MAX_TRACE_ID? can they be reverted?

Reply:
The plan is to remove max trace id since it's misleading. With this change the max trace id is 2**128 -1

Reply:
should we deprecate it right away since it's misleading now? Would be useful to the user to know that it is too